internal/common/forage/payoff.go

```diff
@@ -0,0 +1,40 @@
+package forage
+
+type payoffRuleFuncType = func(input int) (output int)
+
+// DefaultPayoffRuleKey denotes the default payoff for an instance of foraging
+const DefaultPayoffRuleKey = PayoffSquare

```

Comment:
note the documentation style -- gopls will force you to do this :) This is to make sure documentation is consistent and nice docs can be autogenerated